_layouts/deep_learning.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  {% include head.html %}
+  <div class="container-fluid header-holder">
+    <div class="container">
+      <div class="header-container">
+        <a class="header-logo" href="{{ site.url }}{{ site.baseurl }}" aria-label="PyTorch"></a>
+      </div>
+    </div>
+  </div>
+  <body class="{{ page.body-class }}">
+    <div class="main-background {{ page.background-class }}"></div>
+    <div class="jumbotron jumbotron-fluid">
+      <div class="container deep-learning-container"> 
+        <div class="deep-learning-row">
+          <div class="col-md-7 col-8">
+            <h1>Deep Learning with PyTorch</h1>
+            <p class="lead">Download a free copy of the book and learn how to get started with AI / ML development using PyTorch</p>
+          </div>
+          <div class="col-md-5 col-4">
+            <img class="deep-learning-book" src="/assets/images/DeepLearningThumbnail.png">
+          </div> 
+        </div>
+      </div>
+    </div>
+    <div class="main-content-wrapper">
+      <div class="main-content deep-learning-content">
+        <div class="container">
+          <div class="row">
+            {% if page.deep-learning-thank-you == true %}
+              <div class="deep-learning-thank-you-text-contain">
+                <h1 class="deep-learning-thank-you-text">Deep Learning with PyTorch</h1>
```

Comment:
You probably don't need this H1 header on the thank you page since it's just repeating the jumbotron text. The design doesn't even have a body section; it just features that really long jumbotron section. We can see what Bruce thinks of this approach.